public/app/features/admin/ServerStatsCard.tsx

```diff
@@ -3,26 +3,56 @@ import React from 'react';
 import Skeleton from 'react-loading-skeleton';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { CardContainer, Stack, useStyles2 } from '@grafana/ui';
+import { Card, Stack, useStyles2, Tooltip, Icon } from '@grafana/ui';
 
 export interface Props {
-  content: Array<Record<string, string | number | undefined>>;
+  content: Array<Record<string, string | number | boolean | undefined>>;
```

Comment:
we should define the types properly here now if we're going to add this much structure

```tsx
interface StatItem {
  name: string,
  value: number,
  tooltip?: string,
  highlight?: boolean,
  tabbed?: boolean
}

export interface Props {
  content: StatItem[]
  ...
}
```